src/components/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -0,0 +1,314 @@
+<template>
+  <div id="name-translation">
+    <confirm-dialog
+      ref="confirmTranslationDialog"
+      attach="#name-translation"
+    />
+    <v-layout class="mt-3" v-if="!isEditing">
+      <v-flex md3>
```

Comment:
Check your layout with different screen widths (even though we're not trying hard to be mobile-friendly)... in other places I changed from "md" to "xs" to get satisfactory resizing.